### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by nmarley**
* Update package author
* Cleanup README a bit and add publish instructions to README
* Dashify documentation and remove bower references

Based on #13, so that should be merged first.

Please use "Squash and Merge option" when merging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-A module for [dashcore](https://github.com/dashevo/dashcore) that implements [Payment Protocol](https://github.com/bitcoin/bips/blob/master/bip-0070.mediawiki) and other related BIPs.
+A module for [dashcore-lib](https://github.com/dashevo/dashcore-lib) that implements [Payment Protocol](https://github.com/bitcoin/bips/blob/master/bip-0070.mediawiki) and other related BIPs.
 
 ## Getting Started
 
-This library is distributed in both the npm and bower packaging systems.
-
 ```sh
 npm install dashcore-lib
 npm install dashcore-payment-protocol
```
Comment (Cofresi) [inline README.md]:
```
npm install @dashevo/dashcore-lib
npm install @dashevo/dashcore-payment-protocol
```

**Review by Cofresi — CHANGES_REQUESTED** [review]
use the @dashevo namespace npm packages

File: docs/index.md [hunk 2]
```diff
@@ -1,28 +1,22 @@
...
-
-For client-side projects:
+Payment protocol is implemented as a separate module and you must add it to your dependencies:
 
 ```
-bower install bitcore-lib --save
-bower install bitcore-payment-protocol --save
+npm install dashcore-lib
+npm install dashcore-payment-protocol
 ```
```
Comment (Cofresi) [inline docs/index.md:12]:
```
npm install @dashevo/dashcore-lib
npm install @dashevo/dashcore-payment-protocol
```

File: docs/index.md [hunk 3]
```diff
@@ -1,28 +1,22 @@
...
 
 ## Make Payment Details
+
 Here the merchant's server will construct the payment details message:
 
 ```javascript
-var PaymentProtocol = require('bitcore-payment-protocol');
+var PaymentProtocol = require('dashcore-payment-protocol');
 var now = Date.now() / 1000 | 0;
```
Comment (Cofresi) [inline docs/index.md:21]: `var PaymentProtocol = require('@dashevo/dashcore-payment-protocol');`

**Review by Cofresi — CHANGES_REQUESTED** [review]
same as above

File: README.md (hunk 1 above)
Comment (nmarley) [reply README.md]: Nice catch! Should be fixed in both files now.

**Review by Cofresi — APPROVED** [review]

**Pull request merged**